EventHorizon/Indicator/AuraIndicator.lua

```diff
@@ -35,10 +35,15 @@ function AuraIndicator:new(target, start, stop, spellId, texture, config)
 	self.adjustExtendedOffset = 0
 	self.originalDuration = stop - start
 
-	if config.ticks and config.ticks > 0 then		
-		self.numTicks = config.ticks
-		self.originalDuration = stop - start
-		self.snapInterval = self.originalDuration / self.numTicks
+	if config.ticks and config.tickType == "count" and config.tickCount > 0 then		
+		self.numTicks = config.tickCount
+		self.snapInterval = self.originalDuration / self.numTicks		
+	elseif config.ticks and config.tickType == "interval" and config.tickInterval > 0 then
+		self.snapInterval = config.tickInterval
+		self.numTicks = math.floor(self.originalDuration/config.tickInterval)		
+	end
+
+	if config.ticks then
```

Comment:
Since arg `config` is stored in an instance property `self.config` all references afterwards should be to `self.config`

```
	if self.config.ticks and self.config.tickType == "count" and self.config.tickCount > 0 then		
		self.numTicks = self.config.tickCount
		self.snapInterval = self.originalDuration / self.numTicks	
	elseif self.config.ticks and self.config.tickType == "interval" and self.config.tickInterval > 0 then
		self.snapInterval = self.config.tickInterval
		self.numTicks = math.floor(self.originalDuration/self.config.tickInterval)
	end

	if self.config.ticks then
		for i=1,self.numTicks do
			local tick = TickIndicator(target, start + i*self.snapInterval)
			tinsert(self.ticks, tick)
		end
	end
```

Reply:
fixed in latest commit